src/components/AssetsTable/AssetsTable.test.jsx

```diff
@@ -566,13 +563,13 @@ describe('Lock asset', () => {
     });
     const statusAlert = wrapper.find('StatusAlert');
     expect(statusAlert.prop('alertType')).toEqual('danger');
-    expect(statusAlert.find('.alert-dialog').text()).toEqual('Failed to toggle lock for marmoset.png.');
+    expect(statusAlert.find('.alert-dialog').text()).toEqual('Failed to toggle lock for marmoset.png');
```

Comment:
Just flagging an example of the missing punctuation here, which I mentioned above.

Reply:
This one should probably be reverted back to the `.`